src/Exceptionless.Core/Extensions/UriExtensions.cs

```diff
@@ -10,7 +10,7 @@ public static class UriExtensions {
         }
 
         public static string ToQueryString(this IEnumerable<KeyValuePair<string, string>> collection) {
-            return collection.ToConcatenatedString(pair => pair.Key == null ? pair.Value : $"{pair.Key}={pair.Value}", "&");
+            return collection.ToConcatenatedString(pair => pair.Key == null ? pair.Value : $"{pair.Key}={System.Web.HttpUtility.UrlEncode(pair.Value)}", "&");
```

Comment:
I didn't see this in your first commit and apologies for not noticing it earlier. Do you know if any of these methods will work to do this: https://stackoverflow.com/questions/3840762/how-do-you-urlencode-without-using-system-web Using System.Web will be a blocker for us moving to asp.net core later this year.